bokeh/core/_templates/onload_js.js

```diff
@@ -0,0 +1,12 @@
+var id = setInterval(embed_document, 100, window);
```

Comment:
This is a potential solution for embedding bokeh docs in notebooks/nbconvert situations where it's necessary to wait until Bokeh is attached to the window. I don't know if it's better quit trying after some number of intervals, in that case that loading BokehJS has totally failed.

Reply:
If we keep this, I'll dedupe this and DOCS_JS.js

Reply:
Hrm maybe, sure. But 100ms startup for every plot is alot. If a notebook has 100 plots that will not be negligible. would suggest 10ms max. It would maybe be better to split into `embed_document` and a helper that does the render. Then `embed_document` can try to render immediately (by calling the helper), and only if it fails the first attempt, add the helper on an interval.

Reply:
Also def think there should be a max timeout